composer.json

```diff
@@ -72,7 +72,7 @@
         "phpunit/phpunit": "^9.5",
         "rector/rector": "^0.18.2",
         "sylius-labs/coding-standard": "^4.0",
-        "sylius/grid-bundle": "^1.7 || dev-symfony-7",
+        "sylius/grid-bundle": "^1.7 || ^v1.13@alpha",
```

Comment:
I don't think "v" prefix is needed.

Reply:
I think we should bump that grid bundle version cause this one is not maintained anymore.